web/src/components/AppBar/AppBar.vue

```diff
@@ -113,7 +114,7 @@
         rounded
         @click="login"
       >
-        Login
+        {{ DJANGO_API ? 'Login or Create Account' : 'Login' }}
```

Comment:
While I can verify that this works, I'd just like to point out that ideally this would be two different buttons, since they do two different things. You could use a `<template v-if="!DJANGO_API">` above, to catch the create/login buttons for the girder end, and in the `v-else` have the single `Login or Create Account` button.